example/ios/RNPermissionsExample/Info.plist

```diff
@@ -35,6 +35,10 @@
 			</dict>
 		</dict>
 	</dict>
+	<key>NSBonjourServices</key>
+	<array>
+		<string>_lnp._tcp.</string>
+	</array>
```

Comment:
this  may need a separate documentation chunk to avoid a million "how do I use this?" questions :-)

Reply:
Good shout, will add

Reply:
Added in b5701475468a1f89e62b62a2f4ab5991a91691d6